Formula/wget2.rb

```diff
@@ -0,0 +1,67 @@
+class Wget2 < Formula
+  desc "Multithreaded metalink/file/website downloader"
+  homepage "https://gitlab.com/gnuwget/wget2/"
+  url "https://ftp.gnu.org/gnu/wget/wget2-2.0.0.tar.gz"
+  sha256 "4fe2fba0abb653ecc1cc180bea7f04212c17e8fe05c85aaac8baeac4cd241544"
+  license "GPL-3.0-or-later"
+
+  # NOTE: no optional or recommended
+  head do
+    url "https://gitlab.com/gnuwget/wget2.git", branch: "master"
+
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "libtool" => :build
+    depends_on "lzip" => :build
+  end
+
+  # Question: which are required for running and which are build-only?
```

Comment:
You may consult AUR: [wget2](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=wget2), [wget2-git](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=wget2-git).

Reply:
Thanks for suggestion! Will take a look soon.

Reply:
Also, 
[Formula Cookbook](https://docs.brew.sh/Formula-Cookbook#double-check-for-dependencies) recommends otool, but there is also a `brew linkage`

Reply:
It seems that in `wget2` AUR, everything (except doc-related) are requirements and should not be `:build` only. Also `lzip` should be moved out of `head`?
In `wget2-git`, it seems that `rsync` should remain `:build`, also `wget` should be added (?).

Will make those changes in next commit. (Not sure about `wget` bit...)